package.json

```diff
@@ -80,6 +81,7 @@
     "@tiptap/extension-underline": "^2.0.0-beta.195",
     "@tiptap/suggestion": "^2.0.0-beta.195",
     "@tiptap/vue-2": "^2.0.0-beta.195",
+    "lodash.debounce": "^4.0.8",
```

Comment:
What's the reason to use lodash.debounce instead of debounce? As far as I know, we use debounce in many other apps.